admin_manual/enterprise/external_storage/windows-network-drive_configuration.rst

```diff
@@ -144,247 +144,229 @@ ownCloud supports SMB notifications with an ``occ`` command, ``occ wnd:listen``.
 
-.. code-block:: console
+The File Serializer
+-------------------
 
-   chown root /etc/systemd/system/owncloud-listener.service
-   chmod 644 /etc/systemd/system/owncloud-listener.service
+The file serializer is a serializer implementation that can be used with the ``wnd:process-queue`` command.
+It requires an additional parameter where you can specify the location of the file containing the serialized storages. 
 
-Now, you can control your new service like any other, such as using the following command:
+There are several things you should know about this serializer:
 
-.. code-block:: console
+- The generated file contains the encrypted passwords for accessing the backend. This is necessary in order to avoid re-fetching the user information, when next accessing the storages.
+- The generated file is intended to be readable and writeable **only** for the web server user. Other users shouldn't have access to this file. Do not manually edit the file. You can remove the file if it contains obsolete information.
 
-   systemctl start owncloud-listener
- 
-If you’re happy with it, you can configure the script to auto-start on boot, by using the following command.
+Usage Recommendations
+~~~~~~~~~~~~~~~~~~~~~
 
-.. code-block:: console
+Number of Serializers
+^^^^^^^^^^^^^^^^^^^^^
 
-   systemctl enable owncloud-listener.service (or whatever you have named your file).
+Only one file serializer should be used per server + share, as the serialized file has to be per server + share.
+Consider the following usage scenarios:
 
-.. note:: 
-   If you need multiple listeners, just change the name of the file and configure the ``ExecStart`` parameters accordingly.
+- You have several Windows servers. Then you'll need to serialize to different files for each of server.
+- You have three shares: ``10.0.2.2/share1``, ``10.0.2.2/share2``, and ``10.0.10.20/share2``. Then you should use three different calls to ``wnd:process-queue``, changing the target file for the serializer for each one.
 
-.. note::
-   This process is based on `a WND Listener Configuration on ownCloudCentral <https://central.owncloud.org/t/wnd-listener-configuration/3114>`_.
+Since the serialized file has to be per server + share, the serialized file has some checks to prevent misuse. 
+Specifically, if we detect you're trying to read the storages for another server + share from the file, the contents of the file won't be read and will fallback to creating the storage from scratch. 
+We'll then update the contents of that file with the new storage.
 
-.. _password-options-label:
+.. "unneeded competition" needs clarification or rewording
+Note that this will create an unneeded competition, so again, use a different file per server and share.
```

Comment:
What I meant there is that several process-queue will "compete" for the serializer file. Let's say you have 2 process-queue targeting the same serializer file. The first process will create the file, then the second process will notice that the file isn't for it and recreate the file with new content, then the first process runs again and notice the file isn't for it and recreate the file again, and so on.
In this case, the purpose of the serializer file isn't fulfilled